test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -438,10 +439,22 @@ function mapToBids(auctions) {
 
 function getLocalAssert() {
   function isValidAnalyticsReport(report) {
-    assert.containsAllKeys(report, ['analyticsVersion', 'pid', 'src', 'pbjsVersion', 'auctions']);
+    assert.containsAllKeys(report, ['analyticsVersion', 'pid', 'src', 'pbjsVersion', 'auctions', 'gdpr']);
     if ('usPrivacy' in report) {
       assert.match(report.usPrivacy, /[0|1][Y|N|-]{3}/);
     }
+    assert.oneOf(report.gdpr, [0, 1]);
+    if (report.gdpr === 1) {
+      assert.isString(report.gdprConsent);
```

Comment:
I don't think we are actually hitting these new asserts, not seeing unit tests that pass those values to this function.

Reply:
fixed